`npm install --production` fails due to `devDependencies` being needed.

The `prepublish` command does a `npm run test` which needs `karma` and lots of other `devDependencies`, so the following fails like:

```
(nvm  v0.10.40) rye@titan ~/scm/fmd $ git clone git@github.com:FormidableLabs/radium.git radium-test
Cloning into 'radium-test'...
remote: Counting objects: 1886, done.
remote: Compressing objects: 100% (32/32), done.
remote: Total 1886 (delta 13), reused 0 (delta 0), pack-reused 1851
Receiving objects: 100% (1886/1886), 967.97 KiB | 223.00 KiB/s, done.
Resolving deltas: 100% (1024/1024), done.
Checking connectivity... done.
(nvm  v0.10.40) rye@titan ~/scm/fmd $ cd radium-test/
(nvm  v0.10.40) rye@titan ~/scm/fmd/radium-test $ npm install --production
 
> fsevents@0.3.6 install /Users/rye/scm/fmd/radium-test/node_modules/babel/node_modules/chokidar/node_modules/fsevents
> node-gyp rebuild

  SOLINK_MODULE(target) Release/.node
  SOLINK_MODULE(target) Release/.node: Finished
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node
  SOLINK_MODULE(target) Release/fse.node: Finished

> radium@0.13.4 postinstall /Users/rye/scm/fmd/radium-test
> node -e "require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})" || npm run prepublish


> radium@0.13.4 prepublish /Users/rye/scm/fmd/radium-test
> npm test && npm run lint && npm run lib


> radium@0.13.4 test /Users/rye/scm/fmd/radium-test
> karma start

sh: karma: command not found
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0

npm ERR! radium@0.13.4 prepublish: `npm test && npm run lint && npm run lib`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the radium@0.13.4 prepublish script.
npm ERR! This is most likely a problem with the radium package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm test && npm run lint && npm run lib
npm ERR! You can get their info via:
npm ERR!     npm owner ls radium
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 13.4.0
npm ERR! command "/Users/rye/.nvm/v0.10.40/bin/node" "/Users/rye/.nvm/v0.10.40/bin/npm" "run" "prepublish"
npm ERR! cwd /Users/rye/scm/fmd/radium-test
npm ERR! node -v v0.10.40
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/rye/scm/fmd/radium-test/npm-debug.log
npm ERR! not ok code 0

npm ERR! radium@0.13.4 postinstall: `node -e "require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})" || npm run prepublish`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the radium@0.13.4 postinstall script.
npm ERR! This is most likely a problem with the radium package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node -e "require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})" || npm run prepublish
npm ERR! You can get their info via:
npm ERR!     npm owner ls radium
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 13.4.0
npm ERR! command "/Users/rye/.nvm/v0.10.40/bin/node" "/Users/rye/.nvm/v0.10.40/bin/npm" "install" "--production"
npm ERR! cwd /Users/rye/scm/fmd/radium-test
npm ERR! node -v v0.10.40
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/rye/scm/fmd/radium-test/npm-debug.log
npm ERR! not ok code 0
(nvm  v0.10.40) rye@titan ~/scm/fmd/radium-test $ 
```

I _think_ this would be the same result for an application taking a `radium` dependency and then installing the whole app (w/ Radium dependency) with `npm install --production`, which is a very common / sensible use case. (Although I haven't tried out an example with this specifically yet).

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples/",
     "lib": "npm run babel && rimraf lib/__tests__ lib/__mocks__",
     "lint": "eslint modules",
-    "postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || npm run prepublish",
+    "postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || npm run lib",
     "prepublish": "npm test && npm run lint && npm run lib",
     "test": "karma start",
     "test-coverage": "karma start karma.conf.coverage.js"

